model_analyzer/perf_analyzer/perf_config.py

```diff
@@ -295,6 +295,7 @@ def remove_url_from_cli_string(cls, cli_string):
             perf_str_tokens.pop(url_index)
             perf_str_tokens.pop(url_index)
         except ValueError:
+            # Ignore ValueError exception
```

Comment:
Unnecessary comment? (unsure if this is needed for codeql fix)

Reply:
This comment is intended to resolve https://github.com/triton-inference-server/model_analyzer/security/code-scanning/8